### ISSUE TITLE
Compile error- 

### ISSUE THREAD

**Issue by Ang3lERR404** [fig-eater/zig-function-overloading#1]
```
run
└─ run test-presto
   └─ install
      └─ install test-presto
         └─ zig build-exe test-presto Debug native 1 errors
src\main.zig:21:30: error: expected 1 argument(s), found 2
  std.debug.print("{any}", .{tytray(50, 60)});
                             ^~~~~~
C:\Users\{user}\AppData\Local\zig\p\122033a578a940a74e3d1bca5d77fc8c2e6f04403f80427d9aafd4ec6c149afb0b05\src\overloading.zig:101:13: note: function declared here
            fn overloadedfn(args: anytype) ReturnType {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    callMain: C:\zig\lib\std\start.zig:524:32
    WinStartup: C:\zig\lib\std\start.zig:363:45
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
error: the following command failed with 1 compilation errors:
C:\zig\zig.exe build-exe -ODebug --dep overload -Mroot=C:\Users\{user}\Desktop\Folders\coed\zig\testo-presto\src\main.zig -Moverload=C:\Users\{user}\AppData\Local\zig\p\122033a578a940a74e3d1bca5d77fc8c2e6f04403f80427d9aafd4ec6c149afb0b05\src\overloading.zig --cache-dir C:\Users\{user}\Desktop\Folders\coed\zig\testo-presto\.zig-cache --global-cache-dir C:\Users\{user}\AppData\Local\zig --name test-presto --listen=-
Build Summary: 0/5 steps succeeded; 1 failed (disable with --summary none)
run transitive failure
└─ run test-presto transitive failure
   ├─ zig build-exe test-presto Debug native 1 errors
   └─ install transitive failure
      └─ install test-presto transitive failure
         └─ zig build-exe test-presto Debug native (reused)
error: the following build command failed with exit code 1:
C:\Users\{user}\Desktop\Folders\coed\zig\testo-presto\.zig-cache\o\8fdf1ecb7c931100dd42ce240282b8a7\build.exe C:\zig\zig.exe C:\Users\{user}\Desktop\Folders\coed\zig\testo-presto C:\Users\scarl\Desktop\Folders\coed\zig\testo-presto\.zig-cache C:\Users\{user}\AppData\Local\zig --seed 0xafeb7325 -Zce297ceaa9f2d03a run
```
# Was just making a short test

**Comment by fig-eater** [comment]
Sorry, I believe I need to be more specific in the usage. When passing multiple parameters into the overloaded function, pass them within a tuple. I believe the error is happening from that.

So in your code try doing:
`tytray(.{50, 60})`
Instead

**Comment by fig-eater** [comment]
Yeah, I will update the usage. As I originally wrote it incorrectly. Thank you for the catch on that!

**Comment by fig-eater** [comment]
Usage information should be fixed in 9cf5d88d268aa5fade7037e77f770183ad73c18a

If https://github.com/fig-eater/zig-function-overloading/issues/1#issuecomment-2183639259 doesn't fix the problem let me know.

**Comment by Ang3lERR404** [comment]
Heheh, no problem! I'd love to contribute to this little project ^^
